maxValue/reducer.py

```diff
@@ -1,27 +1,24 @@
 import sys
 
+oldKey = None
```

Comment:
Not descriptive at all. You're iterating through different stores, why don't use something like `old_store`?